src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -599,6 +619,15 @@ public void updateFeedbackResponseOptimized(FeedbackResponseAttributes newAttrib
         getPM().close();
     }
     
+    /**
+     * Calls updateFeedbackResponseOptimized with false as the keepUpdateTimestamp parameter
```

Comment:
The comment should tell the reader what the method does (i.e. the result), not how it does it.